charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.9.0"
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 0.9.9
+version: 0.10.0
```

Comment:
Sorry for the confusion. You need to bump the patch and keep minor version the same, ie, 0.9.x.

Reply:
got it